APIBacklog/documentation/MeetingMinutes/MOM-2023-11-02.md

```diff
@@ -0,0 +1,153 @@
+| [#294](https://github.com/camaraproject/WorkingGroups/issues/294) | Verizon| **New API proposal: Network Insights** <br> Updates: <br> - TSC asked specific and understandable API name for Network Insights. Offline discussion with GSMA and Verizon for the new API name for Network Insights. <br> - Verizon: What we have not decided yet is the name for the sub project which will have the specific contribution(yaml) tagged to. i was considering that once the project is kicked off and we have the project call we can finalize the name among the participants interested in this project. If this is holding up the whole process then lets please go ahead with "Connectivity Insights". <br> - API backlog WG will inform TSC that "Connectivity Insights" is proposed as new Sub-Project name (AP 20231019-04 cannot be closed, until TSC ok). <br> - for Sub-Project initial maintainers (AP 20231019-04 can be closed).
+| [#317](https://github.com/camaraproject/WorkingGroups/issues/317) | China Unicom | **New Proposal-Network Slicing API** <br> The API template is available in [PR#316](https://github.com/camaraproject/WorkingGroups/pull/316)<br> Updates: <br> - the API name has been changed from Service Booking to Network Slicing (AP 20231019-05 can be closed). <br> - Jan (Ericsson): he noted that Ericsson supports this API proposal, as it makes sense from technical perspective. Ericsson had comments on the actual API, but before posting them in the thread, he would like to know whether this API is eligible to become a "service API" (and developed in CAMARA) or should be considered as a "Operate API" (and developed in TM Forum). <br> - Eric (Vodafone): he noted that this API should be read as a "Service API" (CAMARA API). He clarified that Vodafone general position for non-OGW related APIs is not to support them. He suggests recommending API owners to follow the default path "OGW Product WS -> CAMARA", in order to get more traction and support from the rest of operators. Finally, he notes that further details on product and technical scopes are needed. <br> - Mark (GSMA): he agreed with most of Eric's comments. Clarified that network slicing APIs are discussed in the context of OPG/OPAG from network enablers perspective. Promotion within Open Gateway Product WS may come in the future, once technical capabilities are available and there is a market demand. <br> - Jose (Telefonica): he agreed with Eric's points, though it also noted that nothing prevents a company from contributing with a non-OGW API. <br> - Eric (Vodafone): he concurred, but also noted that China Unicom is a OGW MoU signatory, and one of the MoU committments is to steer work through OGW programme instead of direct submission to CAMARA. <br> - Jose (Telefonica): suggested keeping discussion off-line, and encouraged all participants to post these comments in the issue/PR thread, so API owners can receive feedback and take action accordingly. |
```

Comment:
The proposal is now in #333, as mentioned in https://github.com/camaraproject/WorkingGroups/issues/317.

Reply:
Good catch. Fixed